pinot-query-planner/src/main/java/org/apache/calcite/rel/rules/PinotAggregateExchangeNodeInsertRule.java

```diff
@@ -111,17 +112,18 @@ public void onMatch(RelOptRuleCall call) {
     ImmutableList<RelHint> oldHints = oldAggRel.getHints();
 
     Aggregate newAgg;
```

Comment:
note to self:
after this main `onMatch` conversion
1. we should check if all the Aggregate functions have desired literals put in place
2. we should make `AggregationFunctionType`s register optional literal checkers on specific operands